src/components/Select/Select.vue

```diff
@@ -596,6 +598,13 @@ export default {
       this.$_focusInner()
       this.hideList()
     },
+
+    /**
+     * emit a load-more event when the user scroll till the end of the list of items
+     */
+    handleInfiniteScroll() {
+      this.$emit('load-more')
```

Comment:
What about this being optional by passing a prop? Maybe users don't want to always execute this.

Reply:
Hi @fariacc thank you &#128522;

Let me ask you, are you saying about emit the event? I thought in to specify a property but I imagined that it would be unnecessary because the user is not required to listen to the event, or what you are thinking is to specify a prop and also change the directive to don't work unless the property  is true? like this 
```js
v-infinite-scroll="{ handler: handleInfiniteScroll, disabled: !infiniteScroll }"
```

Reply:
What I meant was something like this:

```
handleInfiniteScroll() {
  if (this.infiniteScroll) {
    this.$emit('load-more')
  }
}
```

And the `infiniteScroll` prop is false by default. That way, this emit will be executed only when `infinite-scroll` is passed as true